cluster/addons/dns/kube2sky/Changelog

```diff
@@ -1,12 +1,20 @@
-* Fri May 15 2015  Tim Hockin <thockin@google.com>
-  - First Changelog entry
-  - Current version is 1.4
+## Version 1.10 (Jun 19 2015 Tim Hockin <thockin@google.com>)
+- Fall back on service account tokens if no other auth is specified.
 
 
-## Version 1.7 (May 25 2015 Vishnu Kannan <vishnuk@google.com>)
-- Adding support for headless services. All pods backing a headless service is addressible via DNS RR.
+## Version 1.9 (May 28 2015 Abhishek Shah <abshah@google.com>)
```

Comment:
I got the date that was missing from the commits or something.  The exact
dates are not so important.

On Fri, Jun 19, 2015 at 9:24 PM, Robert Bailey <notifications@github.com>
wrote:

> In cluster/addons/dns/kube2sky/Changelog
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10150#discussion_r32882045>
> :
>
> >
> >
> > -## Version 1.7 (May 25 2015 Vishnu Kannan <vishnuk@google.com>)
> > -- Adding support for headless services. All pods backing a headless service is addressible via DNS RR.
> > +## Version 1.9 (May 28 2015 Abhishek Shah <abshah@google.com>)
>
> This version's PR was sent on June 1st and merged on June 5th.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10150/files#r32882045>
> .
>